packages/apollo-link-http/README.md

```diff
@@ -69,7 +69,48 @@ client.query({
 })
 ```
 
-## Upgrading from `apollo-fetch` / `apollo-client` 
+## Errors

```

Comment:
Should we mention `apollo-link-error` in this section as the official way to handle errors?

Reply:
Great thought!